docs/application/native/guides/device/system.md

```diff
@@ -698,7 +714,7 @@ The following table lists the Web feature keys.
 <a name="system"></a>
 ## System Keys
 
-The following table lists the build system keys.
+The following table lists the build system keys:
```

Comment:
Hello.
In line 732, we have a sentence "The following table lists the manufacturer keys." and it looks like colon should comes at the end of this sentence also.

@kkuchciak Would you please update this sentence?

@rishu-samsung When kkuchciak update it, please approve this PR again after checking the latest version.
I'll merge this PR with your approval.

Reply:
Thanks for finding the mistake. I've updated the file.